From e2012b96f50a6b7e05841dfbbee6ecef90c41050 Mon Sep 17 00:00:00 2001
From: Christian Hofstaedtler <christian.hofstaedtler@deduktiva.com>
Date: Mon, 6 Jun 2016 20:59:56 +0200
Subject: [PATCH] Rename notify to pdns_notify to avoid conflicts

Note that this does not change regression-tests.recursor for now,
because that needs a newer pdns-tools deb.

[zeha@debian.org: edited for downstream]
---
 docs/Makefile.am                                       |  2 +-
 docs/manpages/notify.1.md                              | 16 ----------------
 docs/manpages/pdns_notify.1.md                         | 16 ++++++++++++++++
 pdns/Makefile.am                                       | 10 +++++-----
 pdns/notify.cc                                         |  2 +-
 13 files changed, 36 insertions(+), 36 deletions(-)
 delete mode 100644 docs/manpages/notify.1.md
 create mode 100644 docs/manpages/pdns_notify.1.md

diff --git a/docs/Makefile.am b/docs/Makefile.am
index 179a241..0b3ffe3 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -15,7 +15,7 @@ MANPAGES_TARGET_TOOLS = calidns.1 \
 	dnswasher.1 \
 	dumresp.1 \
 	ixplore.1 \
-	notify.1 \
+	pdns_notify.1 \
 	nproxy.1 \
 	nsec3dig.1 \
 	saxfr.1 \
diff --git a/docs/manpages/notify.1.md b/docs/manpages/notify.1.md
deleted file mode 100644
index 166bcca..0000000
--- a/docs/manpages/notify.1.md
+++ /dev/null
@@ -1,16 +0,0 @@
-% NOTIFY(1)
-% PowerDNS.com BV
-% April 2016
-
-# NAME
-**notify** - A simple DNS NOTIFY sender
-
-# SYNOPSIS
-**notify** *IP_ADDRESS*[:*PORT*] *DOMAIN*
-
-# DESCRIPTION
-**notify** sends a DNS NOTIFY message to *IP_ADDRESS*, by default on port 53, for
-*DOMAIN* and prints the remote nameserver's response.
-
-# OPTIONS
-None
diff --git a/docs/manpages/pdns_notify.1.md b/docs/manpages/pdns_notify.1.md
new file mode 100644
index 0000000..f189240
--- /dev/null
+++ b/docs/manpages/pdns_notify.1.md
@@ -0,0 +1,16 @@
+% PDNS_NOTIFY(1)
+% PowerDNS.com BV
+% April 2016
+
+# NAME
+**pdns_notify** - A simple DNS NOTIFY sender
+
+# SYNOPSIS
+**pdns_notify** *IP_ADDRESS*[:*PORT*] *DOMAIN*
+
+# DESCRIPTION
+**pdns_notify** sends a DNS NOTIFY message to *IP_ADDRESS*, by default on port 53, for
+*DOMAIN* and prints the remote nameserver's response.
+
+# OPTIONS
+None
diff --git a/pdns/Makefile.am b/pdns/Makefile.am
index 7332a7c..de08741 100644
--- a/pdns/Makefile.am
+++ b/pdns/Makefile.am
@@ -86,7 +86,7 @@ bin_PROGRAMS += \
 	dnstcpbench \
 	dnswasher \
 	dumresp \
-	notify \
+	pdns_notify \
 	nproxy \
 	nsec3dig \
 	saxfr \
@@ -114,7 +114,7 @@ EXTRA_PROGRAMS = \
 	dumresp \
 	kvresp \
 	ixplore \
-	notify \
+	pdns_notify \
 	nproxy \
 	nsec3dig \
 	saxfr \
@@ -923,7 +923,7 @@ nproxy_LDADD = \
 	$(OPENSSL_LIBS) \
 	$(BOOST_PROGRAM_OPTIONS_LIBS)
 
-notify_SOURCES = \
+pdns_notify_SOURCES = \
 	arguments.cc \
 	base32.cc \
 	base64.cc base64.hh \
@@ -944,12 +944,12 @@ notify_SOURCES = \
 	statbag.cc \
 	unix_utility.cc
 
-notify_LDFLAGS = \
+pdns_notify_LDFLAGS = \
 	$(AM_LDFLAGS) \
 	$(OPENSSL_LDFLAGS) \
 	$(BOOST_PROGRAM_OPTIONS_LDFLAGS)
 
-notify_LDADD = \
+pdns_notify_LDADD = \
 	$(OPENSSL_LIBS) \
 	$(BOOST_PROGRAM_OPTIONS_LIBS)
 
diff --git a/pdns/notify.cc b/pdns/notify.cc
index c07514c..1d7b22f 100644
--- a/pdns/notify.cc
+++ b/pdns/notify.cc
@@ -32,7 +32,7 @@ ArgvMap &arg()
 }
 
 void usage() {
-  cerr<<"Syntax: notify IP_ADDRESS[:PORT] DOMAIN"<<endl;
+  cerr<<"Syntax: pdns_notify IP_ADDRESS[:PORT] DOMAIN"<<endl;
 }
 
 int main(int argc, char** argv)
diff --git a/docs/notify.1 b/docs/notify.1
deleted file mode 100644
index 3c8997b..0000000
--- a/docs/notify.1
+++ /dev/null
@@ -1,17 +0,0 @@
-.TH "NOTIFY" "1" "April 2016" "" ""
-.SH NAME
-.PP
-\f[B]notify\f[] \- A simple DNS NOTIFY sender
-.SH SYNOPSIS
-.PP
-\f[B]notify\f[] \f[I]IP_ADDRESS\f[][:\f[I]PORT\f[]] \f[I]DOMAIN\f[]
-.SH DESCRIPTION
-.PP
-\f[B]notify\f[] sends a DNS NOTIFY message to \f[I]IP_ADDRESS\f[], by
-default on port 53, for \f[I]DOMAIN\f[] and prints the remote
-nameserver\[aq]s response.
-.SH OPTIONS
-.PP
-None
-.SH AUTHORS
-PowerDNS.com BV.
diff --git a/docs/pdns_notify.1 b/docs/pdns_notify.1
new file mode 100644
index 0000000..afc49a3
--- /dev/null
+++ b/docs/pdns_notify.1
@@ -0,0 +1,17 @@
+.TH "PDNS_NOTIFY" "1" "April 2016" "" ""
+.SH NAME
+.PP
+\f[B]pdns_notify\f[] \- A simple DNS NOTIFY sender
+.SH SYNOPSIS
+.PP
+\f[B]pdns_notify\f[] \f[I]IP_ADDRESS\f[][:\f[I]PORT\f[]] \f[I]DOMAIN\f[]
+.SH DESCRIPTION
+.PP
+\f[B]pdns_notify\f[] sends a DNS NOTIFY message to \f[I]IP_ADDRESS\f[], by
+default on port 53, for \f[I]DOMAIN\f[] and prints the remote
+nameserver\[aq]s response.
+.SH OPTIONS
+.PP
+None
+.SH AUTHORS
+PowerDNS.com BV.
